lib/std/fs.zig

```diff
@@ -1490,22 +1490,91 @@ pub const Dir = struct {
         }
     }
 
+    /// Calls makeOpenDirAccessMaskW recursively to make an entire path
```

Comment:
Please avoid recursion in this function. Recursion is problematic because it has unbounded stack usage that potentially depends on user input. It also makes the compiler unable to perform many kinds of optimizations.

Related: #1006

It does not look like you actually used recursion here, so perhaps the only change needed is that this doc comment is incorrect.

Reply:
Thank you for the feedback!
Yes, It isn't an actual recursion. The call stack size is O(1) and doesn't depend on the user input. 
Could you please take care of the incorrect doc comment?

Reply:
@andrewrk note that this 'recursively' language matches the doc comment of `makePath` above which shares the same logic.